### PR TITLE
Add icons for dabpumps custom integration

### PR DESCRIPTION
**Pull request by ankohanse**
Add icons for dabpumps custom integration

## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- Link to integration documentation on our website: https://github.com/ankohanse/hass-dab-pumps

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

File: custom_integrations/dabpumps/icon.png
[the remark this answers is not in the document]
Comment (ankohanse) [reply custom_integrations/dabpumps/icon.png:1]: They are inconsistent with their own icons.
I have taken their own main corporate website as leading: [https://www.dabpumps.com/en](https://www.dabpumps.com/en)

image

[the remark this answers is not in the document]
Comment (frenck) [reply custom_integrations/dabpumps/icon.png:1]: Right that would be the logo, which is correct. The icon, however, they seem to use consistently different compared to what you've submitted in this PR.

[the remark this answers is not in the document]
Comment (ankohanse) [reply custom_integrations/dabpumps/icon.png:1]: Agreed.
Added a different image for the icon. Also improved the quality of the icon and image png and added the 2x size icon and image

**Review by frenck — APPROVED** [review]
Nice! Thanks, @ankohanse 👍 

../Frenck

**Pull request merged**